libraries/client/client_webhook.py

```diff
@@ -187,6 +192,12 @@ def process_webhook(self):
             source_url = self.build_multipart_source(file_key, book)
             identifier, job = self.send_job_request_to_tx_manager(commit_id, source_url, rc, repo_name, repo_owner,
                                                                   count=book_count, part=i, book=book)
+
+            # Send lint request to tx-manager
+            lint_results = self.send_lint_request_to_tx_manager(job, source_url)
```

Comment:
Currently we just create one zip file that contains all the bible books.  It took too much time create 66 individual zip files.  But yes you can do it for each individual book.  Will just need to pass a flag to only unzip that specific book for linting.